### PR TITLE
Fixes #1794. Prevent form submission when pressing enter

### LINKED ISSUE #1794: Enter in description textbox submits form
As described in this bug: https://github.com/webcompat/web-bugs/issues/9706

If you press enter in the description textbox, it submits the form before you've (potentially) finished filling it out.

Maybe we should prevent this behaviour and force the user to click the button?

### PR DESCRIPTION
**Pull request by miketaylr**
...in the description input.

r? @zoepage

### CODE REVIEW

File: webcompat/static/js/lib/bugform.js [hunk 1]
```diff
@@ -67,6 +67,14 @@ function BugForm() {
   this.stepsToReproduceField = this.inputs.steps_reproduce.el;
 
   this.init = function() {
+    // prevent accidental submission with the Enter key
+    // (for now, just the description input)
+    Mousetrap.bind("enter", function(e) {
+      if (e.target.className.indexOf('js-desc') !== -1) {
```
Comment (miketaylr) [inline webcompat/static/js/lib/bugform.js]: oops, prettier will not like those single quotes.

**Comment by miketaylr** [comment]
ping @zoepage

File: webcompat/templates/home-page/form.html [hunk 2]
```diff
@@ -56,7 +56,7 @@ <h3 class="wc-Title--s">Report Site Issue</h3>
               {% endif %}
             </div>
             <span class="wc-Form-validation">
-            {{ form.description(class_='wc-Form-item required',
+            {{ form.description(class_='wc-Form-item required mousetrap js-desc',
```
Comment (zoepage) [inline webcompat/templates/home-page/form.html]: One tiny thing. Would you be able to change `js-desc` to `js-Desc` as this is our naming convention? Thanks!

Comment (miketaylr) [reply webcompat/templates/home-page/form.html]: sure, that works.

**Comment by miketaylr** [comment]
This maybe broke other tests... I didn't bother to check Travis because things were broken when I wrote this. >_<

**Pull request merged**